pyproject.toml

```diff
@@ -125,6 +125,9 @@ line-length = 120
 target-version = ["py39", "py310", "py311", "py312"]
 
 
+[tool.isort]
+profile = "black"
```

Comment:
While `pyproject.toml` is considered as a preferred format of configuration for `isort`, we continue using `setup.cfg` instead for it (and for most of other tools as well) since we think this way it's easier to separate external project characteristics (description, supported language versions, dependencies, etc.) and development tools settings.  
So, please move this to the `setup.cfg`

Reply:
P.S. Having `tool.isort` section here makes `isort` ignore settings in the `setup.cfg`, that's why you have reformatted imports in many project files

Reply:
The reason why I changed the isort format is I faced formatting conflicts around import and pylint comment.

I'm okay to move it to setup.cfg, but many tools has been migrated to pyproject.toml already as much as possible.

This should be a separate topic though.

Reply:
Moved the configuration into setup.cfg  5f3211d